x-pack/plugins/security_solution/public/app/index.tsx

```diff
@@ -5,20 +5,10 @@
  */
 
 import React from 'react';
-import { Store, Action } from 'redux';
 import { render, unmountComponentAtNode } from 'react-dom';
 
-import { AppMountParameters } from '../../../../../src/core/public';
-import { State } from '../common/store';
-import { StartServices } from '../types';
 import { SecurityApp } from './app';
-import { AppFrontendLibs } from '../common/lib/lib';
-
-interface RenderAppProps extends AppFrontendLibs, AppMountParameters {
```

Comment:
I moved this to `x-pack/plugins/security_solution/public/app/types.ts`. This change isn't really necessary.